api/v1alpha1/hardwareclassification_types.go

```diff
@@ -22,41 +22,41 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// HardwareClassificationControllerSpec defines the desired state of HardwareClassificationController
-type HardwareClassificationControllerSpec struct {
+// HardwareClassificationSpec defines the desired state of HardwareClassification
+type HardwareClassificationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of HardwareClassificationController. Edit HardwareClassificationController_types.go to remove/update
+	// Foo is an example field of HardwareClassification. Edit HardwareClassification_types.go to remove/update
 	Foo string `json:"foo,omitempty"`
 }
 
-// HardwareClassificationControllerStatus defines the observed state of HardwareClassificationController
-type HardwareClassificationControllerStatus struct {
+// HardwareClassificationStatus defines the observed state of HardwareClassification
+type HardwareClassificationStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 }
 
 // +kubebuilder:object:root=true
```

Comment:
Shouldn't the status be declared as subresource ? And maybe you could add some short names, since that's a long name to type in kubectl

Reply:
Hey maelk, we are working on introducting short names for most of the field so we will address it in next PR.

Reply:
ok, what about status as subresource ?

Reply:
@Ashughorla @digambar15  Why not declare the status as subresource ? It is assumed in PR 17 https://github.com/metal3-io/hardware-classification-controller/pull/17/files#diff-a683702516d3abc0476a39d02ce7e142R37

Reply:
@maelk We have addressed both changes suggested by you.
- Status under resources
- Short name of hardwareClassification as hwcc and hcc

Reply:
@maelk We have addressed both changes suggested by you in latest commit.
- Status under resources.
- Short name of hardwareClassification as hwcc and hcc